vscode-dotnet-sdk-extension/src/extension.ts

```diff
@@ -99,6 +99,7 @@ export function activate(context: vscode.ExtensionContext, extensionContext?: IE
     };
 
     const timeoutValue = extensionConfiguration.get<number>(configKeys.installTimeoutValue);
+    const finalTimeoutValue = timeoutValue === undefined ? defaultTimeoutValue : timeoutValue;
```

Comment:
This value flows through a different places and at the end of it's journey it's multiplied by 1000 (presumably to generate a number of milliseconds). Could you change the name of these fields to be explicit that this time value in intended to be seconds here (and where the value flows to)?

Reply:
That's a good point, thank you. resolvedTimeoutSeconds is the new name.